mobile/native/src/position.rs

```diff
@@ -0,0 +1,50 @@
+use crate::db;
+use crate::dlc::ChannelState;
+use crate::dlc::DlcChannel;
+use crate::dlc::SignedChannelState;
+use crate::event;
+use crate::event::subscriber::Subscriber;
+use crate::event::EventInternal;
+use crate::event::EventType;
+use trade::ContractSymbol;
+
+#[derive(Clone)]
```

Comment:
🔧 I guess it can be `Copy` 😆 But is this even needed?

Reply:
Yes it's needed for the event hub.

Reply:
It's weird to have to `Clone` and empty type.

Reply:
Its because of this `subscribers: HashMap<EventType, Vec<Box<dyn Subscriber + 'static + Send + Sync>>>,` weirdness.

Reply:
And I think this is needed because we keep the event hub in the storage state.